src/server/src/main/java/io/cassandrareaper/jmx/JmxProxy.java

```diff
@@ -128,4 +128,6 @@ int triggerRepair(
 
   void removeRepairStatusHandler(int repairNo);
 
+  Map<String, List<String>> getTokensByNode();
```

Comment:
Can we move this to `EndpointSnitchInfoProxy` or to a new domain-specific Proxy class please.

Reply:
@michaelsembwever, I made the requested changes and created the `StorageServiceProxy` class.

Reply:
I had avoided `StorageServiceProxy` as it's a "junk yard" where everything gets shoved into. I was kinda hoping for something more "domain" expressive. But it's not a blocker.